src/components/CardProgressSlider.vue

```diff
@@ -0,0 +1,229 @@
+<script lang="ts">
+// libraries
+import { defineComponent, computed } from 'vue';
+import { Screen } from 'quasar';
+
+// types
+import { CardProgress as CardProgressType, ConfigGlobal } from './types';
+
+// config
+const rideToWorkByBikeConfig: ConfigGlobal = JSON.parse(
+  process.env.RIDE_TO_WORK_BY_BIKE_CONFIG
+);
+
+export default defineComponent({
+  name: 'CardProgressSlider',
+  props: {
+    card: {
+      type: Object as () => CardProgressType,
+      required: true,
+    },
+  },
+  setup(props) {
+    const timelineValue = computed(() => {
+      if (!props.card.duration?.current || !props.card.duration?.total) {
+        return 0;
+      }
+      return props.card.duration?.current / props.card.duration?.total;
+    });
+
+    const isMediumScreen = computed(() => {
+      return Screen.gt.xs
+    });
+
+    const isLargeScreen = computed(() => {
+      return Screen.gt.md;
+    });
+
+    // responsive sizing for q-circular-progress
+    const circleSize = computed(() => {
+      let size = '128px';
+      size = isMediumScreen.value ? '180px' : size;
+      size = isLargeScreen.value ? '220px' : size;
+      return size;
+    });
+
+    const borderRadius = rideToWorkByBikeConfig.borderRadiusCard;
+
+    return {
+      timelineValue,
+      circleSize,
+      borderRadius,
+    };
+  },
+});
+</script>
+
+<template>
```

Comment:
If component props is true use simpler declaration e.g. `:dark="true` -> `dark` please.

```vue
<q-card
    dark
    flat
    bordered
    :style="{ 'border-radius': borderRadius }"
    class="bg-blue-grey-3"
    data-cy="card"
  >
```

Reply:
Fixed.